templates/menu.tpl

```diff
@@ -28,42 +30,42 @@
                     {if $authentication_has_role.global_admin}
                         {strip}
                             <li class="dropdown">
-                                <a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
-                                   aria-expanded="false" href="{#url_list_admin#}">{$PALANG.pAdminMenu_list_admin} <span
+                                <a class="btn navbar-btn dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
+                                   aria-expanded="false" href="{#url_list_admin#}"><span class="glyphicon glyphicon-list" aria-hidden="true"></span> {$PALANG.pAdminMenu_list_admin} <span
```

Comment:
These changes (adding icons etc.) also look like something for a separate PR. (Feel free to combine it with the main.tpl changes PR.)

Reply:
probably part of https://github.com/postfixadmin/postfixadmin/pull/604 ?

Reply:
Would have been a good fit, but since #604 is already merged (but didn't include main.tpl and menu.tpl), it's too late ;-)

Reply:
Just found out that these two files to their icons added in #603 :-)

I guess this MR needs a rebase on latest master ;-)